common/src/main/java/dev/latvian/mods/kubejs/item/ItemModificationProperties.java

```diff
@@ -57,4 +82,85 @@ public void setFoodProperties(Consumer<FoodBuilder> consumer) {
 		consumer.accept(builder);
 		item.setFoodPropertiesKJS(builder.build());
 	}
+
+	private Multimap<Attribute, AttributeModifier> mutateAttributeMap() {
```

Comment:
Also, wouldn't this recreate the attributes map *every time* a single modifier is added? That seems... a bit inefficient

Reply:
In ModifiableItemKJS, the mutation now checks if `defaultModifiers`  an instance of `ImmutableMap`, and will skip map creating if it's mutable.